packages/shared/src/components/Settings.tsx

```diff
@@ -109,11 +115,11 @@ export default function Settings({
       </Section>
       <Section>
         <SectionTitle>Preferences</SectionTitle>
-        <div className="flex flex-col items-start pl-1.5 -my-0.5">
+        <SectionContent>
           <Switch
             inputId="hide-read-switch"
             name="hide-read"
-            className="my-3"
+            className="my-3 big"
```

Comment:
What does this big class do, I don't see any reference of that in our styles?

Reply:
I think that is something related to the switch - not quite sure though but all of the switches inside the Settings have that class.

Reply:
I don't see any use for it, could you see if we can remove all of these?
Don't want to have random classes that don't do anything if that makes sense

Reply:
Totally agree. I have instead created a common component since all of the switches are the same on some properties then removed the unnecessary class. Thank you! 🚢